src/applications/mhv-secure-messaging/components/ThreadList/ThreadListItem.jsx

```diff
@@ -101,11 +101,7 @@ const ThreadListItem = props => {
       </div>
       <div className="vads-l-col vads-u-margin-left--1">
         <Link
-          aria-label={`${
-            unreadMessages ? 'Unread message. Subject:' : 'Message subject:'
-          } ${categoryLabel}: ${subject}, ${formattedDate()}. ${
-            hasAttachment ? ' Has attachment.' : ''
-          }`}
```

Comment:
may need to add a child element i.e. `<span class="sr-only">Unread message</span>` to preserve. unread status in aria read out